File: RDCPanel/RDCPanel.Web/Modules/ServerTypes/Employees/EmployeeForm.ts
import { StringEditor, ImageUploadEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface EmployeeForm {
    Name: StringEditor;
    JobTitle: StringEditor;
    Department: StringEditor;
    Extension: StringEditor;
    Email: StringEditor;
    Image: ImageUploadEditor;
    Branch: StringEditor;
    Initial: StringEditor;
}

export class EmployeeForm extends PrefixedContext {
    static readonly formKey = 'Employees.Employee';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!EmployeeForm.init)  {
            EmployeeForm.init = true;

            var w0 = StringEditor;
            var w1 = ImageUploadEditor;

            initFormType(EmployeeForm, [
                'Name', w0,
                'JobTitle', w0,
                'Department', w0,
                'Extension', w0,
                'Email', w0,
                'Image', w1,
                'Branch', w0,
                'Initial', w0
            ]);
        }
    }
}

File: RDCPanel/RDCPanel.Web/Modules/Employees/Employee/EmployeeForm.cs
using Serenity.ComponentModel;

namespace RDCPanel.Employees.Forms;

[FormScript("Employees.Employee")]
[BasedOnRow(typeof(EmployeeRow), CheckNames = true)]
public class EmployeeForm
{
    public string Name { get; set; }
    public string JobTitle { get; set; }
    public string Department { get; set; }
    public string Extension { get; set; }
    public string Email { get; set; }
    public string Image { get; set; }
    public string Branch { get; set; }
    public string Initial { get; set; }
}